src/tools/kdb/specmount.cpp

```diff
@@ -3,7 +3,7 @@
  *
  * @brief source file of spec mount command
  *
- * @copyright BSD License (see LICENSE.md or https://www.libelektra.org)
+ * @Copyright BSD License (see LICENSE.md or https://www.libelektra.org)
```

Comment:
I suspect doxygen will only accept the command lowercase. Either way, all our doxygen commands are lowercase, so let's stick to it.

```suggestion
 * @copyright BSD License (see LICENSE.md or https://www.libelektra.org)
```